internal/output/table.go

```diff
@@ -212,64 +209,63 @@ type tbInnerResponse struct {
-			sourcePath, err := filepath.Rel(workingDir, source.Path)
-			if err == nil { // Simplify the path if possible
-				source.Path = sourcePath
-			}
+	for _, eco := range result.Ecosystems {
+		for _, source := range eco.Sources {
+			for _, pkg := range source.Packages {
+				var everything []VulnResult
 
-			// Ensure that groups are sorted consistently using the first ID in each group
-			slices.SortFunc(pkg.Groups, func(a, b models.GroupInfo) int {
-				return identifiers.IDSortFunc(a.IDs[0], b.IDs[0])
-			})
+				everything = append(everything, pkg.RegularVulns...)
+				everything = append(everything, pkg.HiddenVulns...)
 
-			// Merge groups into the same row
-			for _, group := range pkg.Groups {
-				if !(group.IsCalled() == calledVulns && group.IsGroupUnimportant() == unimportantVulns) {
-					continue
-				}
+				for _, vuln := range everything {
+					outputRow := table.Row{}
+					shouldMerge := false
 
-				outputRow := table.Row{}
-				shouldMerge := false
+					var links []string
+
+					if vuln.VulnAnalysisType != vulnAnalysisType {
+						continue
+					}
 
-				var links []string
+					for _, id := range vuln.GroupIDs {
+						links = append(links, OSVBaseVulnerabilityURL+text.Bold.Sprintf("%s", id))
 
-				for _, vuln := range group.IDs {
-					links = append(links, OSVBaseVulnerabilityURL+text.Bold.Sprintf("%s", vuln))
+						// For container scanning results, if there is a DSA, then skip printing its sub-CVEs.
+						if strings.Split(id, "-")[0] == "DSA" {
+							break
+						}
+					}
 
-					// For container scanning results, if there is a DSA, then skip printing its sub-CVEs.
-					if strings.Split(vuln, "-")[0] == "DSA" {
-						break
+					outputRow = append(outputRow, strings.Join(links, "\n"))
+
+					// todo: this is just to make the snapshots pass without change
+					if vuln.SeverityScore == "N/A" {
+						outputRow = append(outputRow, "")
+					} else {
+						outputRow = append(outputRow, vuln.SeverityScore)
 					}
-				}
 
-				outputRow = append(outputRow, strings.Join(links, "\n"))
-				outputRow = append(outputRow, group.MaxSeverity)
+					outputRow = append(outputRow, eco.Name)
+
+					name := pkg.Name
 
-				if pkg.Package.Ecosystem == "" && pkg.Package.Commit != "" {
-					pkgCommitStr := results.PkgToString(pkg.Package)
-					outputRow = append(outputRow, "GIT", pkgCommitStr, pkgCommitStr)
-					shouldMerge = true
-				} else {
-					name := pkg.Package.Name
 					// TODO(#1646): Migrate this earlier to the result struct directly
-					if depgroups.IsDevGroup(ecosystem.MustParse(pkg.Package.Ecosystem).Ecosystem, pkg.DepGroups) {
+					if depgroups.IsDevGroup(ecosystem.MustParse(eco.Name).Ecosystem, pkg.DepGroups) {
 						name += " (dev)"
 					}
-					outputRow = append(outputRow, pkg.Package.Ecosystem, name, pkg.Package.Version)
+					outputRow = append(outputRow, name)
+					outputRow = append(outputRow, pkg.InstalledVersion)
+					outputRow = append(outputRow, strings.TrimPrefix(source.Name, ":"))
```

Comment:
version here can be git commit, I think we can just use getInstalledVersionOrCommit(pkg)